### DISCUSSION TITLE
Home screen widgets for subreddit

### DISCUSSION THREAD

**Discussion by Maio-msdos** [Docile-Alligator/Infinity-For-Reddit#1051] [Duplicate Ideas] [answered]
Home screen widget which displays the top N posts (N depending on the widget size) of a selected sub reddit

Apollo IOS reddit client does has something similiar

**Comment by Wladefant** [comment]
Before requesting something you should really search for it first

**Reply by Maio-msdos** [reply]
Yes they are both about widgets, but that other request is for a more complex interactive scrolling widget whereas this is simpler and only intended for reporting/displaying?

**Reply by Wladefant** [reply]
Yes, but the topic is the same, you can just start an new thread there. That is the beautiful thing about github discussions

**Comment by Wladefant** [comment, marked as answer]
Duplicate of #917
